src/index.js

```diff
@@ -22,15 +29,18 @@ ReactDOM.render(
         <UserProvider>
           <ProductProvider>
             <CartProvider>
-              <Routes>
-                <Route path="/" element={<App />}>
-                  <Route index element={<Homepage />} />
-                  <Route path="shop" element={<ProductPage />} />
-                  <Route path="login" element={<LoginPage />} />
-                  <Route path="cart" element={<CartPage />} />
-                </Route>
-                <Route path="mock" element={<Mockman />} />
-              </Routes>
+              <WishlistProvider>
+                <Routes>
+                  <Route path="/" element={<App />}>
+                    <Route index element={<Homepage />} />
+                    <Route path="shop" element={<ProductPage />} />
+                    <Route path="login" element={<LoginPage />} />
+                    <Route path="cart" element={<CartPage />} />
+                    <Route path="wishlist" element={<WishListPage />} />
+                  </Route>
+                  <Route path="mock" element={<Mockman />} />
```

Comment:
You can give a null-route if no routes matching.

Reply:
thanks for suggestion. Noted.